.drone.star

```diff
@@ -14,65 +14,89 @@ windowsglobalimage="cppalliance/dronevs2019"
-  linux_cxx("B2_TOOLSET=gcc-5 B2_CXXSTD=11 Job 10", "g++-5", packages="g++-5", buildscript="drone", buildtype="boost", image=linuxglobalimage, environment={'B2_TOOLSET': 'gcc-5', 'B2_CXXSTD': '11', 'DRONE_JOB_UUID': 'b1d5781111'}, globalenv=globalenv),
-  linux_cxx("B2_TOOLSET=gcc-6 B2_CXXSTD=11,14 Job 11", "g++-6", packages="g++-6", buildscript="drone", buildtype="boost", image=linuxglobalimage, environment={'B2_TOOLSET': 'gcc-6', 'B2_CXXSTD': '11,14', 'DRONE_JOB_UUID': '17ba079149'}, globalenv=globalenv),
-  linux_cxx("B2_TOOLSET=gcc-7 B2_CXXSTD=14,17 Job 12", "g++-7", packages="g++-7", buildscript="drone", buildtype="boost", image=linuxglobalimage, environment={'B2_TOOLSET': 'gcc-7', 'B2_CXXSTD': '14,17', 'DRONE_JOB_UUID': '7b52009b64'}, globalenv=globalenv),
-  linux_cxx("B2_TOOLSET=gcc-8 B2_CXXSTD=17 Job 13", "g++-8", packages="g++-8", buildscript="drone", buildtype="boost", image=linuxglobalimage, environment={'B2_TOOLSET': 'gcc-8', 'B2_CXXSTD': '17', 'DRONE_JOB_UUID': 'bd307a3ec3'}, globalenv=globalenv),
-  linux_cxx("B2_TOOLSET=gcc-9 B2_CXXSTD=17 Job 14", "g++-9", packages="g++-9", buildscript="drone", buildtype="boost", image=linuxglobalimage, environment={'B2_TOOLSET': 'gcc-9', 'B2_CXXSTD': '17', 'DRONE_JOB_UUID': 'fa35e19212'}, globalenv=globalenv),
-  linux_cxx("B2_TOOLSET=gcc-10 B2_CXXSTD=17", "g++-10", packages="g++-10", buildscript="drone", buildtype="boost", image=linuxglobalimage, environment={'B2_TOOLSET': 'gcc-10', 'B2_CXXFLAGS': '-funsigned-char', 'B2_CXXSTD': '17', 'DRONE_JOB_UUID': 'fa35e19213'}, globalenv=globalenv),
-  linux_cxx("B2_TOOLSET=gcc-11 B2_CXXSTD=17", "g++-11", packages="g++-11", buildscript="drone", buildtype="boost", image=linuxglobalimage, environment={'B2_TOOLSET': 'gcc-11', 'B2_CXXFLAGS': '-funsigned-char', 'B2_CXXSTD': '17,2a'}, globalenv=globalenv),
-  linux_cxx("B2_TOOLSET=gcc-12 B2_CXXSTD=17", "g++-12", packages="g++-12", buildscript="drone", buildtype="boost", image="cppalliance/droneubuntu2204:1", environment={'B2_TOOLSET': 'gcc-12', 'B2_CXXFLAGS': '-funsigned-char', 'B2_CXXSTD': '17,20'}, globalenv=globalenv),
+  # Latest compilers
+  linux_cxx("GCC 12: C++17", "g++-12", packages="g++-12", buildscript="drone", buildtype="boost", image="cppalliance/droneubuntu2204:1", environment={'B2_TOOLSET': 'gcc-12', 'B2_CXXFLAGS': '-funsigned-char', 'B2_CXXSTD': '17,20'}, globalenv=globalenv),
+  linux_cxx("Clang 14: C++11, 14, 17, 20", "clang++-14", packages="clang-14 libstdc++-10-dev", llvm_os="jammy", llvm_ver="14", buildscript="drone", buildtype="boost", image="cppalliance/droneubuntu2204:1", environment={'B2_TOOLSET': 'clang-14', 'B2_CXXSTD': '11,14,17,20'}, globalenv=globalenv),
+  windows_cxx("MSVC 14.3: C++17, 20", "", image="cppalliance/dronevs2022", buildtype="boost", buildscript="drone", environment={"B2_TOOLSET": "msvc-14.3", "B2_CXXSTD": "17,20"},globalenv=globalenv),
```

Comment:
It isn't "Latest compilers" it is
```
  # latest gcc: 11,14,17,20
  # latest clang: 11,14,17,20
```

and there's no MSVC, look at the chart (this is why we pasted it)